car_test.go

```diff
@@ -163,11 +163,6 @@ func TestBadHeaders(t *testing.T) {
 			"{version:\"1\",roots:[baeaaaa3bmjrq]}",
 			"1da265726f6f747381d82a4800010000036162636776657273696f6e6131",
 			"", "invalid header: ",
-		}, {
-			"{version:1}",
-			"0aa16776657273696f6e01",
-			"empty car, no roots",
-			"",
```

Comment:
This can't be removed, we need to have test for both root present and no root, to guard the behavior. Restored in 67151fcd9b1989cd2d1212caccbceca2255b8da1.